src/user/room.ts

```diff
@@ -544,9 +544,9 @@ export class Room extends Accessory implements Sayable {
   }
 
   public emit (event: 'invite', inviter: Contact,         invitation: RoomInvitation)           : boolean
-  public emit (event: 'leave',  leaverList:   Contact[],  remover?: Contact)                    : boolean
-  public emit (event: 'join',   inviteeList:  Contact[],  inviter:  Contact)                    : boolean
-  public emit (event: 'topic',  topic:        string,     oldTopic: string,   changer: Contact) : boolean
+  public emit (event: 'leave',  leaverList:   Contact[],  remover:  Contact, time?: Date)                    : boolean
```

Comment:
The `time` should be optional for `emit` because we must make sure if there's some consumer want to get the time, they will.

Remove `?` to make sure the `time` will be passed.

And I also feel that the `time` should rename to `date` because it's a `Date`.

Reply:
actually it is a `datetime`, shall we rename it as `datetime`?